packages/app/src/specs/InlineSpecList.cy.tsx

```diff
@@ -28,108 +27,154 @@ describe('InlineSpecList', () => {
     },
   })
 
-  it('should render a list of specs', () => {
-    mountInlineSpecList()
-    cy.get('li')
-    .should('exist')
-    .and('have.length', 7)
-
-    // overflow is required for the virtual list to work
-    // this test will fail if the overflow set by `useVirtualList`
-    // is overridden
-    cy.get('[data-cy="specs-list-container"]')
-    .should('have.css', 'overflow-y', 'auto')
+  describe('with no saved search term', () => {
```

Comment:
Most of the changes here are just nesting  according to whether a filter was saved or not.